test/compact/IDEA_withDRC_o1_v03.xml

```diff
@@ -43,13 +43,13 @@
   <include ref="../../FCCee/IDEA/compact/IDEA_o1_v03/LumiCal_o1_v01.xml"/>
 
   <!-- Import vertex -->
-  <include ref="../../FCCee/IDEA/compact/IDEA_o1_v03/Vertex_IDEA_o1_v01.xml"/>
+  <include ref="../../FCCee/IDEA/compact/IDEA_o1_v03/VertexComplete_IDEA_o1_v03.xml"/>
 
   <!-- Import drift chamber -->
   <include ref="../../FCCee/IDEA/compact/IDEA_o1_v03/DriftChamber_o1_v02.xml"/>
 
   <!-- Import silicon wrapper -->
-  <!-- <include ref="SiliconWrapper_o1_v01.xml"/> -->
+  <include ref="SiliconWrapper_o1_v03.xml"/>
```

Comment:
You are missing the relative path here, this test still fails because of that. By the way, you can try the test before hand with `ctest` from the build directory.

Reply:
Yeah, just found that. Sorry, next time I'll first try it locally